src/uproot/streamers.py

```diff
@@ -815,42 +777,34 @@ def class_code(
 
         if streamerinfo.name == "TBranch" and self.name == "fBasketSeek":
             read_members.append(
-                "        if context.get('speedbump', True):\n"
-                "            if cursor.bytes(chunk, 1, context)[0] == 2:\n"
-                "                tmp = numpy.dtype('>i8')"
+                """        if context.get('speedbump', True):
+                            if cursor.bytes(chunk, 1, context)[0] == 2:
+                                tmp = numpy.dtype('>i8')"""
             )
             read_member_n.append("    " + read_members[-1].replace("\n", "\n    "))
 
         else:
             read_members.append(
-                "        if context.get('speedbump', True):\n"
-                "            cursor.skip(1)"
+                """        if context.get('speedbump', True):
+                               cursor.skip(1)"""
             )
             read_member_n.append("    " + read_members[-1].replace("\n", "\n    "))
 
         read_members.append(
-            "        self._members[{}] = cursor.array(chunk, self.member({}), "
-            "tmp, context)".format(repr(self.name), repr(self.count_name))
+            f"        self._members[{repr(self.name)}] = cursor.array(chunk, self.member({repr(self.count_name)}), tmp, context)"
         )
         read_member_n.append("    " + read_members[-1])
 
         strided_interpretation.append(
-            "        raise uproot.interpretation.objects.CannotBeStrided("
-            "'class members defined by {} of type {} in member "
-            "{} of class {}')".format(
-                type(self).__name__, self.typename, self.name, streamerinfo.name
-            )
+            f"        raise uproot.interpretation.objects.CannotBeStrided('class members defined by {type(self).__name__} of type {self.typename} in member {self.name} of class {streamerinfo.name}')"
         )
 
         awkward_form.extend(
             [
-                "        contents[{}] = ListOffsetForm(context['index_format'], "
-                "uproot._util.awkward_form(cls._dtype{}, file, context),".format(
-                    repr(self.name), len(dtypes)
-                ),
-                "            parameters={'uproot': {'as': 'TStreamerBasicPointer', "
-                "'count_name': " + repr(self.count_name) + "}}",
-                "        )",
+                f"        contents[{repr(self.name)}] = ListOffsetForm(context['index_format'], uproot._util.awkward_form(cls._dtype{len(dtypes)}, file, context),",
+                "            parameters={'uproot': {'as': 'TStreamerBasicPointer', 'count_name': "
+                + repr(self.count_name)
+                + "}},)",
             ]
         )
```

Comment:
This one is a bit convoluted: the `repr(self.count_name)` is still concatenated with `+`, though it's an f-string. The number of curly brackets gets exciting because you need to fill some code inside JSON. And then the whole things is an `awkward_form.extend` with a list of two strings: the `extend` + comma is doing the same work that concatenation would do. Maybe it was so that the JSON wouldn't be part of the f-string, so that the curly brackets wouldn't have to be doubled?